sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/Storage/BlobsCheckpointStore.cs

```diff
@@ -15,7 +15,7 @@ namespace Azure.Messaging.EventHubs.Processor
     ///   A storage blob service that keeps track of checkpoints and ownership.
     /// </summary>
     ///
-    public sealed class BlobsCheckpointStore : PartitionManager
+    internal sealed class BlobsCheckpointStore : PartitionManager
```

Comment:
Yes, we probably can.   If we do make these public at some point, we'll probably keep a `partial` class approach to expose those constructors internally, so it should be safe.   I'll defer to @kinelski on the plan for this.   

Since this isn't a public surface change, we can always follow-up after the release on this one.

Reply:
Yes, we can remove it. I'll try to include it in my open PR.